### ISSUE TITLE
export swapping adjective and adjective satellite in sense_key

### ISSUE THREAD

**Issue by alexandretessarollo** [own-pt/mill#50]
I used mill export over OWN-EN files and it worked fine for most cases. When it deals with adjectives, though, it seems to change the classification of _adjective_ and _adjective satellite_ in the sense-key. This changes significantly the sense_key, mainly due to the effect of head_word and head_id.

This effect on sense_key results in more difficulty in mapping synset_ids between different versions of OWN (both EN and PT).

**Comment by arademaker** [comment]
can you provide some examples?

**Comment by odanoburu** [comment]
> can you provide some examples?

yes, please! specially since I haven't been working on this recently I need some help remembering details (:

also, which export [backend](https://github.com/own-pt/mill/wiki/Export) did you use? it's easier if you include the command you ran!

**Comment by arademaker** [comment]
@odanoburu the ideia is that @alexandretessarollo made many changes in the own-en and we want to evaluate the impact of those changes in the WSD of 1-2 corpora. We fixed the algorithm for WSD, it is UKB. The challenge is how to compare the outputs of UKB with the different versions of OWN-EN.

**Comment by odanoburu** [comment]
thanks, I get the context now. I'll wait for the examples!
